spec/workflow/states/wait_spec.rb

```diff
@@ -0,0 +1,22 @@
+RSpec.describe Floe::Workflow::States::Pass do
+  let(:workflow) { Floe::Workflow.load(GEM_ROOT.join("examples/workflow.asl")) }
+  let(:state)    { workflow.states_by_name["WaitState"] }
+
+  describe "#run!" do
+    it "sleeps for the requested amount of time" do
+      allow(state).to receive(:sleep).with(state.seconds)
```

Comment:
This we should do an `expect` here.  If we need it to _actually_ wait, then we can tack on `.and_call_original`

```suggestion
      expect(state).to receive(:sleep).with(state.seconds)
```

Reply:
:+1: good catch, done